test/stdlib/TestCharacterSet.swift

```diff
@@ -289,8 +289,17 @@ class TestCharacterSet : TestCharacterSetSuper {
             expectNotEqual(abcSet, abcdSet)
             
             expectTrue(abcSet.isStrictSubset(of:abcdSet))
-            expectFalse(abcdSet.isStrictSubset(of:abcSet))
-            
+            #if (arch(i386) || arch(x86_64)) && os(iOS)
+            let vers = ProcessInfo.processInfo.operatingSystemVersion
+            if vers.majorVersion >= 10 && vers.minorVersion >= 3 && vers.patchVersion >= 0 {
+                expectFalse(abcdSet.isStrictSubset(of:abcSet))
+            } else {
+                print("running on \(vers) but was guarded to 10.3")
```

Comment:
I agree - however it looks like the simulator is either picking up the wrong version of CoreFoundation or the guard is not working correctly. I was able to verify on the iOS hardware build and the macOS build that this code works correctly - leaving one of two options; either a) the mastering of the simulator was somehow broken (which is very scary) or the linkage is somehow picking up the host OS CoreFoundation instead of the simulator version (still somewhat scary but has happened before in some screwy cases)